utils/cli/core/definition.py

```diff
@@ -112,6 +112,12 @@ def argparse(s):
     type=parse_label_arg,
     help='string or file containing JSON labels specification'
 )
+task_create_parser.add_argument(
+    '--project_id',
```

Comment:
@adamhrv , thanks for the contribution. Could you please rename `--project_id` option to just `--project`?